spec/jobs/bulk_invite_spec.rb

```diff
@@ -100,8 +100,8 @@
       expect(User.where(staged: true).find_by_email('test@discourse.org')).to eq(nil)
       expect(user.user_fields[user_field.id.to_s]).to eq('value 1')
       expect(user.user_fields[user_field_color.id.to_s]).to eq('Blue')
-      expect(staged_user.user_fields[user_field.id.to_s]).to eq('value 2')
-      expect(staged_user.user_fields[user_field_color.id.to_s]).to eq(nil)
+      expect(staged_user.reload.user_fields[user_field.id.to_s]).to eq('value 2')
```

Comment:
Nitpicky but do we need to reload twice?

Reply:
Indeed that’s not necessary, that’s what happens when you just copy/paste stuff :sweat_smile: